src/Stratis.BitcoinD/Stratis.BitcoinD.csproj

```diff
@@ -14,6 +14,8 @@
 
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
     <NoWarn>1701;1702;1705;IDE0008;</NoWarn>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
```

Comment:
I'm unsure about this - I remember your chat with @noescape00  about this - are we committing this one to master ?

Reply:
you still need to revert this file right ?